boards/native/nrf_bsim/CMakeLists.txt

```diff
@@ -15,6 +15,15 @@ zephyr_compile_options(
 # as both the HW models and embedded SW use them.
 target_compile_options(native_simulator INTERFACE -fshort-enums)
 
+# nrf_bsim is used to simulate ARM devices.
```

Comment:
sidenote: We may have RISCV cores soon too :) (you don't need to change anything)